packages/server/test/util/factories.ts

```diff
@@ -54,6 +54,6 @@ export const QuestionFactory = new Factory(QuestionModel)
   .sequence('text', (i) => `question ${i}`)
   .attr('status', 'Queued')
   .attr('questionType', QuestionType.Other)
-  .attr('createdAt', new Date('2020-03-01T05:00:00.000Z'))
```

Comment:
this breaks some integration tests. We can either put this back, or pass `createdAt` to the factories in the failing specs (might just be queue.integration)

Reply:
Good catch. I added passed a `createdAt` time to the factory to fix the issue